dev/com.ibm.ws.logging/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -62,8 +62,8 @@ trace.enhanced=Use the enhanced basic trace format.
 trace.advanced=Use the advanced trace format.
 
 isoDateFormat=Use the ISO 8601 date format
-isoDateFormat.desc=The date and time use a locale-specific format or the ISO-8601 format. You can specify true or false for the value of the attribute or the value of the equivalent property. The default value is false.�Avoid trouble: Use the com.ibm.ws.logging.isoDateFormat property, except in cases where you are trying to change the configuration dynamically after startup.
-If you specify a value of true, the ISO-8601 format is used in the messages.log file, the trace.log file, and the FFDC logs. The format is yyyy-MM-dd'T'HH:mm:ss.SSSZ.
+isoDateFormat.desc=The date and time use a locale-specific format or the ISO-8601 format. You can specify true or false for the value of the attribute or the value of the equivalent property. The default value is false. Avoid trouble: Use the com.ibm.ws.logging.isoDateFormat property, except in cases where you are trying to change the configuration dynamically after startup. \
```

Comment:
Suggestion for line 65: The date and time use a locale-specific format or the ISO-8601 format. You can specify true or false for the value of the attribute or the value of the equivalent property. The default value is false. Unless you are trying to change the configuration dynamically after startup, use the com.ibm.ws.logging.isoDateFormat property to avoid trouble.

Reply:
I think the "avoid trouble:..." is used in various other places in this file -- so it's used in line 65 the same way for consistency.